app/controllers/action_items_controller.rb

```diff
@@ -40,4 +40,10 @@ def reopen
       redirect_to action_items_path, alert: @action_item.errors.full_messages.join(', ')
     end
   end
+
+  private
+
+  def user_action_items
+    ActionItem.all.where(assignee_id: current_user.id)
```

Comment:
let's use relation from user to get action items 
and scope to find them without status